src/test/resources/org/mule/module/apikit/metadata/extended-api-in-raml08/examples/fixture-get-example.json

```diff
@@ -0,0 +1,5 @@
+              [{
```

Comment:
all this examples can be removed, in order to reduce the api spec

Reply:
Reduced. Just kept the api that I wrote for this case.